src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/ProjectSnapshotManagerBase.cs

```diff
@@ -3,16 +3,25 @@
 
 using System;
 using System.Collections.Immutable;
+using System.Diagnostics.CodeAnalysis;
 using Microsoft.AspNetCore.Razor.ProjectSystem;
 using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.CodeAnalysis.Razor.ProjectSystem;
 
-internal abstract class ProjectSnapshotManagerBase : ProjectSnapshotManager
+internal abstract class ProjectSnapshotManagerBase : IProjectSnapshotManager
```

Comment:
ℹ️ I will be getting rid of `ProjectSnapshotManagerBase` in a later change. I figured it was a bit too much for this PR.